infra/README.md

```diff
@@ -0,0 +1,49 @@
+# GitHub Runner Infrastructure
+
+To redeploy and update GitHub variables, make sure you are logged into AWS and
```

Comment:
I think this part could be slightly clearer about what is needed when. I would suggest something like starting by what is the trigger that needs to do additional work (as opposed to just let the CI do its stuff), and then order the guide by steps:

> If you make any change to the infrastructure code located in this directory,
> you will have to redeploy it. To do that, you'll need to update GitHub variables as well. Do the following:
>
> 1. Make sure you are logged into AWS. You can check that using `awscli2`:
>
> ```console
> nix run nixpkgs#awscli2 -- sts get-caller-identity
>{
>  # CENSORED
>}
>```
>
> If not, log in with AWS SSO credentials, follow [their guide][aws-sso-guide].
>
> 2. Make sure your a logged into GitHub.  You need the requisite permissions
on the Nickel repository to proceed further [..].
> 3. Update the infrastructure:
>   ```console
>   nix develop ..#infra -c update-infra
>   ```